pkg/drivers/powermax.go

```diff
@@ -94,17 +94,23 @@ func PrecheckPowerMax(ctx context.Context, cr *csmv1.ContainerStorageModule, ope
 			Value: "/var/lib/kubelet",
 		})
 	}
+	version, err := utils.GetLatestVersion(string(csmv1.ReverseProxy), operatorConfig)
```

Comment:
If the default config version is cosidered as the latest one, then can the same be considered for a minimal sample file?

Reply:
If the user does not mention any version then the latest version is picked up by default. This is to reduce number of lines in the yaml.